Cargo.toml

```diff
@@ -35,7 +35,7 @@ tracing-error = "0.2.0"
 tracing-subscriber = { version = "0.3.16", features = ["env-filter"] }
 whoami = "1"
 
-subspace-sdk = { git = "https://github.com/subspace/subspace-sdk", rev = "ae2c1f6bd975f3471ee0d7243f2753b10593e194" }
+subspace-sdk = { git = "https://github.com/subspace/subspace-sdk", rev = "dfa0c919368bc0beab9dff5c552a771318eddd92" }
```

Comment:
It is best to stick to commits in `gemini-3d-maintenance` branch for now, as CLI will stop working otherwise

Reply:
ah yeah, thanks for the catch